package.json

```diff
@@ -76,6 +76,7 @@
     "react": "15.4.2",
     "react-addons-test-utils": "15.4.2",
     "react-dom": "15.4.2",
+    "react-intl": "2.2.3",
```

Comment:
Why do we need this in the root? I think this would be part of terra-i18n, the webpack plugin, or terra-site